docs/releases/2.1.txt

```diff
@@ -287,6 +287,9 @@ how to remove usage of these features.
 
 * The ``host`` parameter of ``django.utils.http.is_safe_url()`` is removed.
 
+* ``allowed_hosts`` is now a required argument of
```

Comment:
This goes in the "Backwards-incompatible changes" section rather than "Features removed".